README.md

```diff
@@ -152,6 +152,23 @@ post_install do |installer|
 end
 ```
 
+If you use new Xcode build system and show the alert "Target Pods cannot link framework Foundation.framework", prefere use this `post_install`
+
+```ruby
+post_install do |installer|
+  podsTargets = installer.pods_project.targets.find_all { |target| target.name.start_with?('Pods') }
+  podsTargets.each do |target|
+    target.frameworks_build_phase.clear
+    target.build_configurations.each do |config|
+      if config.name == 'Debug'
+        config.build_settings['OTHER_SWIFT_FLAGS'] = ['$(inherited)', '-Onone']
```

Comment:
These flags are deprecated, in recent Xcode versions `SWIFT_COMPILATION_MODE=wholemodule` in combination with `SWIFT_OPTIMIZATION_LEVEL=-O` can be used

Reply:
Nice!
https://download.developer.apple.com/Developer_Tools/Xcode_10_beta_3/Release_Notes_for_Xcode_10_beta_3.pdf (page 36)